lib/cli/behaviour/template/.eslintignore

```diff
@@ -0,0 +1,37 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
+node_modules
+jspm_packages
+
+# Optional npm cache directory
+.npm
+
+# Optional REPL history
+.node_repl_history
```

Comment:
Does this need to exist?

Other behaviours have minimal .eslintignore files - https://github.com/UKHomeOfficeForms/hof-behaviour-session/blob/master/.eslintignore

Edit: I've just noticed this is copied from hof-behaviour-summary-page, which I added. I think that was a mistake.